auxx/src/AuxxOptions.hs

```diff
@@ -75,14 +76,18 @@ startModeParser :: Parser AuxxStartMode
 startModeParser = option startModeReader $
     long "mode" <>
     value Automatic <>
-    helpDoc (Just $
-           text "Auxx start mode. Can be 'light', 'with-config', 'with-node', 'auto'." <> line
-        <> text "* 'light' mode requires no configuration to start, but the available."
-        <> text " command set is limited." <> line
-        <> text "* 'with-node' mode will start auxx as a node plugin." <> line
-        <> text "* 'with-config' mode will start auxx as a standalone application." <> line
-        <> text "* 'auto' will try to load the configuration and start in 'with-config' mode,"
-        <> text " if fails to do so will start in 'light' mode. (default: auto)")
+    helpDoc (Just $ mconcat
+        [ text "Auxx start mode. Can be 'light', 'with-config', 'with-node', 'auto'."
+        , line
+        , text "* 'light' mode requires no configuration to start, but the available."
+        , text " command set is limited."
+        , line
+        , text "* 'with-node' mode will start auxx as a node plugin."
+        , line
+        , text "* 'with-config' mode will start auxx as a standalone application."
+        , line
+        , text "* 'auto' will try to load the configuration and start in 'with-config' mode,"
+        , text " if fails to do so will start in 'light' mode. (default: auto)" ])
```

Comment:
This is not related to PR goal but personally I think that multiline strings should be written with some interpolation package, like, `neat-interpolation`. So you can just write:

```haskell
   help [text|
        Auxx start mode. Can be 'light', 'with-config', 'with-node', 'auto'.
        * 'light' mode requires no configuration to start, but the available.
          command set is limited.
        * 'with-node' mode will start auxx as a node plugin.
        * 'with-config' mode will start auxx as a standalone application.
        * 'auto' will try to load the configuration and start in 'with-config' mode,
          if fails to do so will start in 'light' mode. (default: auto)
        |]
```